### PR TITLE
Create growth.md

### PR DESCRIPTION
**Pull request by EstebanDalelR**
A doc on how we do growth, set for 2024.

### CODE REVIEW

File: growth.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+### Newsletter 
+
+No current tactics other than the monthly update.
+
+### 𝕏
+This platform suffers from the brevity of it's reach. An amazing post will trend up a few hours, and can trend for any number of reasons. The current voice of the platform is kind of acid, usually mocking our team.
+
+### LinkedIn
+
```
Comment (baristaGeek) [inline growth.md:46]: - The algorithm rewards silence for a while (apparently)
- Great for company announcements
- Ads are expensive and don't convert well
- Great idea to tag people
- Link on the coments
- Better than Twitter for reach, worse than Twitter for real engagement

File: growth.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+No tactics right now.
+### Youtube
+
+Abandoned platform.
+
+### TikTok
+
+Testing short content seems to work. Very personal from the author.
```
Comment (baristaGeek) [inline growth.md:54]: Add ### Google Ads section

File: growth.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Growth
+Growth is a multimodal, very brittle thing. Every time we start focusing on something, all the other channels suffer, but ultimately, making the company grow is one of the main target of the whole organization.
```
Comment (baristaGeek) [inline growth.md]: Remove "very brittle things... channels suffer". Sounds pessimistic, nor it's true.

In fact, success in one channel helps push sucess in another one. Eg: A blog post (content marketing) brings both traffic from Reddit and HN (social media), plus organic search (SEO).

**Review by baristaGeek — CHANGES_REQUESTED** [review]
Very good! I have small comments. But overall very good!

File: growth.md (hunk 3 above)
Comment (EstebanDalelR) [reply growth.md]: Thanks!
Point being pushing people to be careful but I get the vibe.

File: growth.md (hunk 2 above)
Comment (EstebanDalelR) [reply growth.md:54]: Thanks! 
Yes

File: growth.md (hunk 3 above)
Comment (baristaGeek) [reply growth.md]: Ahhh gotcha

**Pull request closed without merging**